doc/rule-descriptions.md

```diff
@@ -37,7 +37,7 @@
 | [dlitem](https://dequeuniversity.com/rules/axe/4.7/dlitem?application=RuleDescription)                                           | Ensures &lt;dt&gt; and &lt;dd&gt; elements are contained by a &lt;dl&gt;                                                                              | Serious                  | cat.structure, wcag2a, wcag131, EN-301-549, EN-9.1.3.1                                                                             | failure                    |                                                                                                                                                                                                                                                                    |
 | [document-title](https://dequeuniversity.com/rules/axe/4.7/document-title?application=RuleDescription)                           | Ensures each HTML document contains a non-empty &lt;title&gt; element                                                                                 | Serious                  | cat.text-alternatives, wcag2a, wcag242, TTv5, TT12.a, EN-301-549, EN-9.2.4.2, ACT                                                  | failure                    | [2779a5](https://act-rules.github.io/rules/2779a5)                                                                                                                                                                                                                 |
-| [duplicate-id-aria](https://dequeuniversity.com/rules/axe/4.7/duplicate-id-aria?application=RuleDescription)                     | Ensures every id attribute value used in ARIA and in labels is unique                                                                                 | Critical                 | cat.parsing, wcag2a, wcag411, EN-301-549, EN-9.4.1.1                                                                               | failure                    | [3ea0c8](https://act-rules.github.io/rules/3ea0c8)                                                                                                                                                                                                                 |
+| [duplicate-id-aria](https://dequeuniversity.com/rules/axe/4.7/duplicate-id-aria?application=RuleDescription)                     | Ensures every id attribute value used in ARIA and in labels is unique                                                                                 | Critical                 | cat.parsing, wcag2a, wcag412, EN-301-549, EN-9.4.1.2                                                                               | needs&nbsp;review          | [3ea0c8](https://act-rules.github.io/rules/3ea0c8)                                                                                                                                                                                                                 |
```

Comment:
Are we still happy with `Critical` for this rule?

Reply:
Yeah, I think that's right. Getting the ID wrong on something like a form label is potentially a vey big problem.